app/assets/javascripts/components/tag_groups/ColorPicker.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import reactCSS from 'reactcss';
+import { TwitterPicker } from 'react-color';
+
+export default class ColorPick extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayColorPicker: false,
+      color: this.props.color
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+
+    this.handleClose = this.handleClick.bind(this);
```

Comment:
The problem isn't the `this`, I omitted it just to be quicker, the problem is that the method to be bounded here is `handleClose`, not `handleClick`.